helidon-ls-ext/helidon.jdt.ext/helidon.jdt.ext.test/jars/.gitignore

```diff
@@ -0,0 +1 @@
+!*.jar
```

Comment:
Checking-in binaries in git is a NO-NO, use a proper Maven practice.

Reply:
This jars used in tests

Reply:
The fact that it used in test is not a valid reason for checking in a JAR file.